### PR TITLE
Create chirp component

### PR DESCRIPTION
**Pull request by DylanBerbette**
## Description

- Adds create chirp testing.
- Wasn't sure how to receive input from chirp box as it wasn't working for me, so I just got rid of it.

- Addresses issue #135

### CODE REVIEW

**Comment by JustinASmith** [comment]
@DylanBerbette This test isn't really doing much. Are you going to add more?

**Comment by DylanBerbette** [comment]
Well I was trying to make it to where it would test the input of the chirp, but I keep getting an error because my chirp input couldn't be received.

**Comment by DylanBerbette** [comment]
I had this line in there, but is wasnt working
expect(chirpInput.value).toBe("TEST TWEET");

**Comment by JustinASmith** [comment]
> I had this line in there, but is wasnt working
> expect(chirpInput.value).toBe("TEST TWEET");

@DylanBerbette Can you post information? Maybe a screenshot and code that produces error

**Comment by JustinASmith** [comment]
> I had this line in there, but is wasnt working
> expect(chirpInput.value).toBe("TEST TWEET");

You aren't setting it

**Comment by JustinASmith** [comment]
@The props passed mention nothing about "content"

**Comment by DylanBerbette** [comment]
image

**Comment by JustinASmith** [comment]
Look at line #31

**Comment by JustinASmith** [comment]
It is chirpData.

**Comment by JustinASmith** [comment]
@DylanBerbette And you can't have a change function change it with fireEvent.change unless you pass your own function (setChirpData) as a prop when rendering. And inside that you'd have to set it. I recommend you just changing the content prop and not worry about the function as a prop.

**Comment by DylanBerbette** [comment]
Im a little confused, but are you saying the content prop should be chirpData?

File: client/src/components/ChirpInputForm/ChirpInputForm.js [hunk 1]
```diff
@@ -24,13 +24,15 @@ const ChirpInputForm = (props) => {
               }
             >
               <FormControl
+                data-testid="chirp-input"
                 placeholder="Say something..."
                 className="input"
                 onChange={e => props.setChirpData(e.target.value)}
                 value={props.chirpData}
```
Comment (JustinASmith) [inline client/src/components/ChirpInputForm/ChirpInputForm.js:31]: > Im a little confused, but are you saying the content prop should be chirpData?

@DylanBerbette  Yes

**Comment by JustinASmith** [comment]
> Im a little confused, but are you saying the content prop should be chirpData?

@DylanBerbette Yes. Look at my comment on files changed tab

**Comment by DylanBerbette** [comment]
Ok im actually so dumb. I think I see what you mean now. So I just changed content to chirpData, is that all I need to do though because im still getting that error.

**Comment by JustinASmith** [comment]
@DylanBerbette I should of been more detailed in my reply. You don't pass a data prop. There is no reference to "props.data" in the component. The only references are "props.chirpData", "props.shareChirp", and "props.setChirpData". The props you set is chirpData, but that is not inside of a data object like you have

**Pull request merged**